qan-api2/migrations/sql/22_pg_stat_monitor_23.down.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE metrics
+  DROP COLUMN m_wal_buffers_full_cnt,
+  DROP COLUMN m_wal_buffers_full_sum,
+  DROP COLUMN m_parallel_workers_to_launch_cnt,
+  DROP COLUMN m_parallel_workers_to_launch_sum,
+  DROP COLUMN m_parallel_workers_launched_cnt,
+  DROP COLUMN m_parallel_workers_launched_sum;
```

Comment:
Just small thing. It is missing empty line at the end. Same for up file.

Reply:
Added